benchmarks/inference-server/text-generation-inference/sample-terraform.tfvars

```diff
@@ -9,13 +9,19 @@ gpu_count = 1
 
 # How to (horizontally) scale the workload. Allowed values are:
 # - null (no scaling),
-# - "cpu" (scale on cpu utilization).
+# - Workload resources:
+#   - "cpu" (scale on cpu utilization).
+# - Workload metrics (i.e. custom metrics):
+#   - "tgi_queue_size"
+#   - "tgi_batch_current_size"
+#   - "tgi_batch_current_max_tokens"
 # - Other possibilities coming soon...
 #
-# Note: "cpu" scaling is a poor choice for this workload - the tgi workload
-# starts up, pulls the model weights, and then spends a minute or two worth of
-# cpu time crunching some numbers. This causes hpa to add a replica, which then
-# spends more cpu time, which causes hpa to add a replica, etc. Eventually,
-# things settle, and hpa scales down the replicas. This whole process could
-# take up to an hour.
+# See `autoscaling.md` for more details and recommendations.
 hpa_type = null
```

Comment:
Default null case throws an error:

kubernetes_manifest.default["./manifest-templates/text-generation-inference-svc.tftpl"]: Refreshing state...
│ Error: Invalid function argument
│ 
│   on main.tf line 27, in locals:
│   25:     contains(
│   26:       ["tgi_queue_size", "tgi_batch_current_size", "tgi_batch_current_max_tokens"],
│   27:       var.hpa_type
│   28:     )
│     ├────────────────
│     │ while calling contains(list, value)
│     │ var.hpa_type is null
│ 
│ Invalid value for "value" parameter: argument must not be null.
╵